netbout-web/src/main/java/com/netbout/rest/TkStart.java

```diff
@@ -58,6 +58,7 @@
  *  this task citing a NullPointerException (See qulice issue #608 raised for
  *  the same) and then remove TkStart.java from qulice exceptions.
  */
+@SuppressWarnings("PMD.ConstructorShouldDoInitialization")
```

Comment:
@danff it should be possible to refactor, please read http://www.yegor256.com/2015/05/07/ctors-must-be-code-free.html